src/services/StorageService/index.ts

```diff
@@ -0,0 +1,82 @@
+import localforage from 'localforage'
+
+import { allCards } from '../../game/cards'
+import { GameStateCorruptError } from '../../game/services/Rules/errors'
+import { ICard } from '../../game/types'
+
+export enum StorageKey {
+  PLAYER_DECK = 'PLAYER_DECK',
+}
+
+export type DeserializedDeck = Map<ICard, number>
+export type SerializedDeck = Record<string, number>
```

Comment:
@jules, use these types where appropriate in `src/services/StorageService/index.test.ts` (such as for the test `deck` data and similar).